### PR TITLE
[release-4.8] vendor: Update PAO dependencies

### PR DESCRIPTION
**Pull request by shajmakh**
- Latency bug fixes 1986587,1989531
    - Netqueues tests enhancements
    - New function added to nodes.go to get tuned pod for node
    - Bug fixes in performance.go: 2013147, 2013216, 2012962, 2004538. And rcu test 42696 enhancement.

### CODE REVIEW

**Comment by Tal-or** [comment]
@shajmakh 
```
Files TESTLIST.md and _cache/TESTLIST.md.old differ
Docs should be regenerated and updated upstream. You can use cnf-tests/hack/fill-empty-docs.sh and cnf-tests/hack/generate-cnf-docs.sh
/go/src/github.com/openshift-kni/cnf-features-deploy
make: *** [Makefile:119: validate-test-list] Error 1
```

File: cnf-tests/docgen/e2e.json [hunk 1]
```diff
@@ -135,4 +140,4 @@
     "sctp Test Connectivity Connectivity between client and server connect a client pod to a server pod via Service Node Port Default namespace": "Pod to pod connectivity via service nodeport, default namespace",
     "xt_u32 Negative - xt_u32 disabled Should NOT create an iptable rule": "Negative test: when the xt_u32 module is not enabled, appling an iptables rule that utilize the module should fail.",
     "xt_u32 Validate the module is enabled and works Should create an iptables rule inside a pod that has the module enabled": "Verifies that an iptables rule that utilize xt_u32 module can be applied successfully in a pod that has the module enabled."
-}
+}
```
Comment (Tal-or) [inline cnf-tests/docgen/e2e.json:143]: New line is missing

Comment (shajmakh) [reply cnf-tests/docgen/e2e.json]: Thanks for letting me know! fixed

**Comment by fedepaol** [comment]
merging as it's a dependency bump
/approve
/lgtm

**Pull request merged**